client/src/plugins/usage-statistics/event-handlers/__tests__/DiagramOpenEventHandlerSpec.js

```diff
@@ -207,6 +213,248 @@ describe('<DiagramOpenEventHandler>', () => {
       diagramType: 'cmmn'
     });
   });
+
```

Comment:
I'd personally create tests for the dedicated metrics ( a - process variables, b - user task forms) and then only test for those. Rationale: we shouldn't be forced to touch the tests whenever we add a new metric, only adding new tests for the new type of metrics.

So what I mean is something like

```js
describe('element templates, () => {

  it('should send element templates');


  it('should resend minimal data if payload is too big');

});


describe('diagram metrics - process variables', () => {  

  it('should send');
  

  it('should send for multiple participants');


  it('should send none');

});


describe('diagram metrics - user tasks', () => {

  it('should send - root level user tasks');


  it('should send - pools');


  it('should send - sub processes');


  it('should send - empty');
});
```

And then for a single test, to only test one thing at a time

```js
it('should send - root level user tasks', async () => {
  
// ...
  const metrics = onSend.getCall(0).args[0].diagramMetrics;

  const { userTask } = metrics.tasks;

  // then
  expect(userTask).to.equal(...);
  
});
```

Reply:
Good point, I'll update that